runtime/src/network/audited.rs

```diff
@@ -25,7 +26,8 @@ impl<S: crate::Sink> crate::Sink for Sink<S> {
 
         self.auditor.event(b"send_success", |hasher| {
             hasher.update(self.remote_addr.to_string().as_bytes());
-            hasher.update(data);
+            // TODO danlaine: do we need this?
+            // hasher.update(data);
```

Comment:
This ensures data moving over the wire is identical (a property we'd like to enforce). Could be a config option?

Reply:
I re-added the data to the audit event but unfortunately this comes with the cost of doing a copy, at least with the existing functions from tokio-uring. I'll investigate whether there's a way to do this without copying, and without using `unsafe` to "manually" create a `&[u8]`.

Reply:
Settled on doing this with no copy using `crate::deref`